pkg/cmd/operator/cmd.go

```diff
@@ -3,6 +3,7 @@ package operator
 import (
 	"fmt"
 
+	"github.com/scylladb/scylla-operator/pkg/cmd/operator/serveprobes"
```

Comment:
isn't the naming off? Looking at other command names it should be called `probeserver` or other similar non-verb

Reply:
renamed